mgmt/rest/client/client_func_test.go

```diff
@@ -46,11 +47,11 @@ var (
 	LOG_LEVEL = log.FatalLevel
 
 	SNAP_PATH               = os.ExpandEnv(os.Getenv("SNAP_PATH"))
```

Comment:
Same thing here -- Do we need SNAP_PATH if all file paths are going through helper?